lib/index.tsx

```diff
@@ -78,6 +77,7 @@ const PROPS_THAT_SHOULD_CAUSE_RERENDER: Array<keyof RecaptchaBaseConfig> = [
 
 class Reaptcha extends Component<Props, State> {
   container?: HTMLDivElement | null;
+  timer?: number | undefined;
```

Comment:
When the property is optional (with `?`), it's automatically `type | undefined`, so the explicit `| undefined` is not needed. But it's just cosmetics.